### ISSUE TITLE
Build Examples Fails 

### ISSUE THREAD

**Issue by Jesse-Millwood** [nicokoch/secret-rs#8]
I tried building the examples with `cargo build --examples` and get the following error: 

```
error[E0432]: unresolved import `glib::ffi::gsize`
 --> secret-sys/src/lib.rs:6:49
  |
6 | use glib::ffi::{GError, GType, GList, gboolean, gsize, gpointer, GHashTable};
  |                                                 ^^^^^ no `gsize` in `ffi`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: Could not compile `secret-sys`.

```

I looked at the glib crate and I don't see `gsize`. Is there a known change for this? Thanks.

**Comment by nicokoch** [comment]
Hello,
I strongly suggest to not use this library any more. it is unmaintained and basically a dead project.
I started this project years ago as my first dive into rust. Since than so many things have changed, that a complete rewrite of this library is the only sensible thing to do.

Apologies for the bad news.

**Comment by Jesse-Millwood** [comment]
Thanks for the reply! 
I'll look into other methods then.

**Comment by Jesse-Millwood** [comment]
Thanks for the reply! 
I'll look into other methods then.

**Comment by penguin359** [comment]
This issue should be fixed with PR #10.
